src/actions/extract/Extract.ts

```diff
@@ -0,0 +1,68 @@
+import QualifierValue from "../../qualifier/QualifierValue";
+import Qualifier from "../../qualifier/Qualifier";
+import Action from "../Action";
+
+/**
+ * @doc
+ * @memberOf Actions
+ * @namespace PSDTools
+ */
+class Extract extends Action {
+  private qualifierValue = new QualifierValue();
+
+  constructor() {
+    super();
+    this.qualifierValue.delimiter = ';';
+  }
+
+  /**
+   * @description Delivers an image containing only specified layers of a Photoshop image.
+   * <b>Learn more:</b> {@link https://cloudinary.com/documentation/paged_and_layered_media#deliver_selected_layers_of_a_psd_image | Deliver selected layers of a PSD image}
```

Comment:
Maybe change docstring to Extracts? instead of delivers

Reply:
Changed